internal/services/automation/automation_webhook_resource_test.go

```diff
@@ -151,12 +151,16 @@ resource "azurerm_automation_webhook" "test" {
   automation_account_name = azurerm_automation_account.test.name
   expiry_time             = "%s"
   runbook_name            = azurerm_automation_runbook.test.name
+
+  lifecycle {
+    ignore_changes = [uri]
+  }
```

Comment:
Just double checking here since our understanding is that for `ignore_changes` to work the Create and Update methods need to be separated, but is this working as expected since this resource is still using a combined `CreateUpdate`?

Reply:
had another look and I think you're right that ignore_changes doesn't make sense - I added a note here instead, thanks!